machine_management/manual_machine_management/user_infra/adding-compute-user-infra-general.adoc

```diff
@@ -0,0 +1,42 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="adding-compute-user-infra-general"]
+= Adding compute machines to clusters with user-provisioned infrastructure manually
+include::_attributes/common-attributes.adoc[]
+:context: adding-compute-user-infra-general
+
```

Comment:
This assembly is subsequently overwhelming and underwhelming! 
The heading after heading after heading seems like a bit much visually, to me. And then, each module is only a link is a bit disappointing. 
I wonder if there is another way to present this.  Or not. Up to you. Matter of opinion.

Reply:
Yeah, sprucing this assembly up is part of the plan. For now it has moved as-is, but your feedback is right on the money in terms of what I think it needs :clinking_glasses: